docs/includes/introduction.txt

```diff
@@ -17,210 +17,211 @@ Using the Extension
 ===================
 
 Usage and installation instructions for this extension are available
-from the `Celery documentation`_:
-
-http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#using-custom-scheduler-classes
-
-
-.. _`Celery documentation`:
-    http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#using-custom-scheduler-classes
+from the :ref:`Celery documentation <beat-custom-schedulers>`.
 
 Important Warning about Time Zones
 ==================================
 
 .. warning::
 
-    If you change the Django ``TIME_ZONE`` setting your periodic task schedule
+    If you change the Django :setting:`TIME_ZONE` setting your periodic task schedule
     will still be based on the old timezone.
 
     To fix that you would have to reset the "last run time" for each periodic
-    task::
+    task:
 
-        >>> from django_celery_beat.models import PeriodicTask, PeriodicTasks
-        >>> PeriodicTask.objects.all().update(last_run_at=None)
-        >>> PeriodicTasks.changed()
+       >>> from django_celery_beat.models import PeriodicTask, PeriodicTasks
+       >>> PeriodicTask.objects.all().update(last_run_at=None)
```

Comment:
PeriodicTask.objects.all().update(last_run_at=None), this could be PeriodicTask.objects.update(last_run_at=None) as per latest django dev docs update